rinja_derive/src/generator.rs

```diff
@@ -122,17 +123,17 @@ impl<'a> Generator<'a> {
         buf.writeln("::Result::Ok(())");
         buf.writeln("}");
 
-        buf.writeln("const EXTENSION: ::std::option::Option<&'static ::std::primitive::str> = ");
-        buf.writeln(&format!("{:?}", self.input.extension()));
-        buf.writeln(";");
-
-        buf.writeln("const SIZE_HINT: ::std::primitive::usize = ");
-        buf.writeln(&format!("{size_hint}"));
-        buf.writeln(";");
-
-        buf.writeln("const MIME_TYPE: &'static ::std::primitive::str = ");
-        buf.writeln(&format!("{:?}", &self.input.mime_type));
-        buf.writeln(";");
+        buf.writeln(format_args!(
```

Comment:
Couldn't it be replaced with the [`writeln` macro](https://doc.rust-lang.org/nightly/std/macro.writeln.html)?

Reply:
[`format_args_nl!()`](https://doc.rust-lang.org/nightly/std/macro.format_args_nl.html) is not stable, yet.

Reply:
Ah, I think I misunderstood you. Answering in the next comment.